modules/navier_stokes/doc/content/source/hdgbcs/NavierStokesHDGOutflowBC.md

```diff
@@ -0,0 +1,17 @@
+# NavierStokesHDGOutflowBC
+
+This class implements an outflow boundary condition for use with the
+hybridized kernel [NavierStokesHDGKernel.md]. The condition imposed is
+
+\begin{equation}
+\vec{n}p -\nu\nabla\vec{u}\cdot{n} = \vec{0}
+\end{equation}
+
+where $\vec{n}$ is the outward-facing normal on the boundary, $p$ is the
+pressure, $\nu$ is the kinematic viscosity, and $\vec{u}$ is the velocity.
```

Comment:
ugh all the fluid properties use mu (dynamic). can you use mu?

Reply:
Are you sure it is the kinematic viscosity? In some incompressible simulations people divide by the constant rho and then this is true but then p is not the real pressure.

Reply:
@lindsayad

Reply:
maybe we should just have KinematicViscosityFunctorMaterial ? and we ll use that for HDG?

Reply:
The current form is divided through by rho. I will change it tho

Reply:
changed